### PR TITLE
Add password option to the diawi upload step

### PR DESCRIPTION
**Pull request by deanWombourne**

### CODE REVIEW

**Review by kceb — APPROVED** [review]
LGTM, thank you for this!

File: step.sh [hunk 1]
```diff
@@ -3,7 +3,23 @@
...
+  cmd="${cmd} -p ${password}"
+fi
+
+# Add the required arguments
+cmd="${cmd} ${api_token} ${filename}"
+
+echo "Running command '${cmd}'"
+
+# Run the command
+diawi_url=$(cmd)
```
Comment (kceb) [inline step.sh]: running this locally it complains `cmd: command not found`

**Comment by kceb** [comment]
Added a fix to this current PR branch:

https://github.com/deanWombourne/bitrise-step-diawi-upload/pull/6

**Comment by deanWombourne** [comment]
FYI @kceb You probably already know this, but this release was merged into the diawi step repo a few days ago so should be available now.

**Pull request merged**